vue.config.js

```diff
@@ -50,6 +50,10 @@ module.exports = {
         target: 'http://localhost:1337',
         changeOrigin: true,
       },
+      // Required for the health response to work on the client
+      '/health': {
+        target: 'http://localhost:1337',
+      },
```

Comment:
Fixes the local health path so it actually works.

Reply:
Resolves #181